src/sdk/namespace_cache.js

```diff
@@ -4,6 +4,7 @@
 const _ = require('lodash');
 const stream = require('stream');
 const assert = require('assert');
+const process = require('process');
```

Comment:
If I recall correctly, we don't need to require process as it is a native global object